.gitlab-ci.yml

```diff
@@ -870,6 +871,14 @@ plugin:ci-serapi:
   needs:
   - build:edge+flambda
   - library:ci-mathcomp
+  only: *full-ci
+
+plugin:ci-coq_lsp:
+  extends: .ci-template-flambda
+  needs:
+  - build:edge+flambda
+  - plugin:ci-serapi
```

Comment:
```suggestion
  - library:ci-mathcomp
  - plugin:ci-serapi
```

Reply:
coq-lsp test suite doesn't need math-comp, so I tried to be more economical.

Reply:
The CI is going to do `make ci-coq_lsp`, which transitively depends on mathcomp in Makefile.ci, so it's going to build mathcomp.